code/modelCuration/deleteInactivatedReactions.m

```diff
@@ -0,0 +1,84 @@
+%
+% FILE NAME:    deleteInactivatedReactions.m
+% 
+% PURPOSE: This script performs a full removal ("hard deletion") of
+%          inactivated (LB = UB = 0) reactions. These reactions were
+%          previously identified as invalid.
+%
+%          The exceptions are the alternative biomass reactions, which will
+%          remain inactivated but will not be removed from the model.
+%
+%              biomass_components
+%              biomass_Recon3D
+%              biomass_maintenance_Recon3D
+%              biomass_maintenance_noTrTr_Recon3D
+%              biomass_HMR_RenalCancer
+%
+
+
+%% Process Human-GEM model
+
+% load model
+ihuman = importHumanYaml('../../modelFiles/yml/HumanGEM.yml');
```

Comment:
It might be good to update the path here ..

Reply:
Generally, this and other one-time-use scripts are not updated with changes in the repository. Since this was how the path was named when the script was created and executed, it seems a bit weird to update it when it will not (and should not) be used again.

That being said, it's a super easy change to make so I'm pretty indifferent.